Composer/packages/extensions/visual-designer/src/widgets/ActionCard.tsx

```diff
@@ -47,8 +48,9 @@ export const ActionCard: WidgetComponent<ActionCardProps> = ({
 }) => {
   const header = disableSDKTitle ? title : generateSDKTitle(data, title);
   const nodeColors = { themeColor: colors.theme, iconColor: colors.icon };
+  const CardRender = data.$type === SDKTypes.ChoiceInput ? ListOverviewCard : FormCard;
```

Comment:
don't couple schema logic here. you can edit the uischema file

Reply:
Now that we already have a new widget, ActionCard.tsx doesn't need to be touched